src/viz-parole-population/VizParolePopulation.js

```diff
@@ -0,0 +1,73 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+import { TOTAL_KEY } from "../constants";
+import StateDistrictMap from "../state-district-map";
+
+const VizParolePopulationContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+`;
+const ParoleViz = styled.figure`
+  margin: 0;
+  flex: 0 1 auto;
+`;
+const MapCaption = styled.figcaption`
+  color: ${(props) => props.theme.colors.body};
+  font: ${(props) => props.theme.fonts.body};
+`;
+
+export default function VizParolePopulation({
+  data: { populationDemographics, districtOffices },
+  districtId,
+  onDistrictClick,
+}) {
+  const districtTotals = populationDemographics
+    .filter(
+      (record) =>
+        record.race_or_ethnicity === TOTAL_KEY &&
+        record.gender === TOTAL_KEY &&
+        record.age_bucket === TOTAL_KEY
+    )
+    .map((record) => {
+      const districtData = districtOffices.find(
+        (office) => +office.district === +record.district
+      );
+      if (districtData) {
+        return {
+          district: `${record.district}`,
+          lat: districtData.lat,
+          long: districtData.long,
+          value: +record.total_supervision_count,
+        };
+      }
+      return null;
+    })
+    .filter((record) => record);
```

Comment:
Why is this filter statement needed?

Reply:
this will filter out any `null`s that may have resulted the previous `map` result